webapp/web/configuration_form.css

```diff
@@ -2,64 +2,103 @@
     display: grid;
     grid-template-rows: auto;
     grid-gap: 20px;
-    border: 1px solid #ddd;
-    border-radius: .5rem;
     padding: 20px;
 }
 
-.single-form-group {
-    display: flex;
-    padding: 27px 0px 27px 10px;
-    border-radius: .5rem;
-    box-shadow: 0px 0px 3px rgba(0, 0, 0, 0.22);
+.form-group {
+  display: grid;
+  grid-gap: 10px;
 }
 
-.single-form-group--in-multi {
-    border: none;
-    border-radius: none;
-    box-shadow: none;
-    padding: 10px 0px;
+.form-group__label {
+  margin-bottom: 10px;
+  font-weight: 500;
+  white-space: nowrap;
 }
 
-.single-form-group__label {
-    align-self: center;
-    padding-right: 10px;
-    font-size: 1.1em;
+.form-group-item {
+  display: grid;
+  grid-template-columns: 25% 40%;
+  margin-left: 20px;
 }
 
-.single-form-group__input {
-    align-self: center;
-    width: 18px;
-    height: 18px;
-    margin-right: 10px;
+.form-group-item--shrink {
+  grid-template-columns: 25px auto;
+  margin: 0px;
 }
 
-.single-form-group__input--is-text {
-    align-self: center;
-    width: 30%;
-    height: 25px;
+.form-group-item--col4 {
+  grid-template-columns: 3% 20% auto auto;
+  grid-gap: 10px;
+  margin: 0px;
 }
 
-.multi-form-group {
-    display: grid;
-    grid-template-rows: auto;
-    grid-gap: 10px;
+.form-group-item__label {
+  white-space: nowrap;
```

Comment:
I think it would be good to wrap the label on two lines or more if the window is too narrow - otherwise it overlaps badly with the second column. Or am I missing something?
```suggestion
```